packages/app/src/app/pages/Dashboard/Content/CreateNewSandbox/Modal/index.js

```diff
@@ -93,12 +92,27 @@ export default class Modal extends React.PureComponent {
           ))}
 
           <Title>Server Templates</Title>
+          {mainServerRows.map((ts, i) => (
+            // eslint-disable-next-line react/no-array-index-key
```

Comment:
You should not use indexes for keys. This can negatively impact performance and may cause issues with state.

refs
https://reactjs.org/docs/lists-and-keys.html#keys
https://github.com/facebook/react/issues/1342#issuecomment-39230939
https://medium.com/@robinpokorny/index-as-a-key-is-an-anti-pattern-e0349aece318

Reply:
Yeah, I know. But if someone isolate and compose this part of code, may affect it. 
I always do this to ensure it will work regardless of context